packages/shared-state-pirania/files/etc/pirania/hooks/start/01-publish_and_sync

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+/etc/shared-state/publishers/shared-state-publish_vouchers
+shared-state sync pirania
```

Comment:
new line at the end of the file is missin

Reply:
done in 9c23f93ecfabab041139f534d3d731e88b9398b9